### PR TITLE
doc: Update Configurator

### PR DESCRIPTION
**Pull request by amyreye**
- Add Configurator Debian package build steps that were removed from GSG
- Add description of error icon on parameters tabs
- Mention that working folder name should be meaningful

Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by amyreye** [comment]
Please comment on whether the build steps are in a logical place.

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Review by gvancuts — APPROVED** [review]
LGTM!

**Review by dbkinder — APPROVED** [review]
LGTM too, thanks

File: doc/tutorials/acrn_configurator_tool.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ Capabilities:
   XML format
```
Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:18]: Would it be useful to add a note about where the board configuration file comes from to line 15?

File: doc/tutorials/acrn_configurator_tool.rst [hunk 2]
```diff
@@ -73,7 +79,8 @@ such as a board configuration file, scenario configuration file, and launch
 scripts. If it finds any, it will delete them.
```
Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:79]: Please add borders to images in the final version.  This is mostly an issue with images that are white where it is difficult to visually see where the image ends.

File: doc/tutorials/acrn_configurator_tool.rst [hunk 3]
```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
       :align: center
```
Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:335]: Line 177 or 184 .  "when you need to iterate your"   I recommend changing "iterate" to "change".  I didn't quite understand what iterate meant in that context and change seems more appropriate.

Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:335]: Lines 244/245  or 251/252  consider changing "scenario configuration files for a former release may not work for a latter if they are not upgraded." to "scenario configuration files for a former release may not work in the current release unless they are upgraded."

File: doc/tutorials/acrn_configurator_tool.rst [hunk 4]
```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
...
       :class: drop-shadow
```
Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:336]: line 286 ish -- when defining basic parameters, should a bullet be added that indicates something like.  - parameters that are anticipated to be commonly used for typical ACRN use cases

File: doc/tutorials/acrn_configurator_tool.rst [hunk 5]
```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
...
```
Comment (cjbooth) [inline doc/tutorials/acrn_configurator_tool.rst:337]: Line 300 ish  -- I think adding the default values note should be on the first bullet as performance optimization will require specification.
* Parameters that are optional for ACRN configuration, compilation, and
  execution. Default values cover most use cases.
* Parameters that are used for fine-grained tuning, such as reducing code
  lines or optimizing performance.

File: doc/tutorials/acrn_configurator_tool.rst (hunk 2 above)
Comment (amyreye) [reply doc/tutorials/acrn_configurator_tool.rst:79]: The published version has a border. https://projectacrn.github.io/latest/tutorials/acrn_configurator_tool.html

Comment (dbkinder) [reply doc/tutorials/acrn_configurator_tool.rst:79]: If you're looking at the github rendering it won't have a drop shadow (as amy mentioned).  I added an image class that we can use to add a drop shadow when the picture would benefit (such a screen shot with a white background):
```
 .. image:: images/configurator-newscenario01.png
          :align: center
          :class: drop-shadow
```

**Pull request merged**